Content.Server/Medical/SuitSensors/SuitSensorSystem.cs

```diff
@@ -256,8 +258,10 @@ private void OnInsert(EntityUid uid, SuitSensorComponent component, EntGotInsert
         if (args.Container.ID != component.ActivationContainer)
             return;
 
-        if (HasComp<DisableSuitSensorsComponent>(args.Container.Owner)) // Frontier: entities with disabled suit sensors must never be set as a valid user.
-            return; // Frontier
+        // Frontier: entities with "Always" disabled suit sensors must never be set as a valid user.
+        if (TryComp<DisableSuitSensorComponent>(args.Container.Owner, out var disableSuitSensor) && disableSuitSensor.Always)
+            return;
```

Comment:
```suggestion
        if (TryComp<DisableSuitSensorsComponent>(args.Container.Owner, out var disableSuitSensor) && disableSuitSensor.RemoveRegistration)
            return;
```
rename consistently vs. component